### PR TITLE
Update PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR updates the PR template

**Review:**
- [x] code approved by PG, AJ
- [x] "Merge" approved by AJ, HS

This is patch **version bump** because it changes only instruction

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-1. install on stage of the coffee machine: `bash servers/resources/coffee.selecta.se/update-cg-stage.sh fix-water-leak`
-1. activate stage: `us`
-1. run following command: `cg get coffee`
+### How to prepare for test
+- [ ] ssh to hasta (depending on type of change)
+- [ ] install on stage:
+`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`
+- [ ] ssh to clinical-db (depending on type of change)
+- [ ] install on stage:
+`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`
```
Comment (jemten) [inline .github/PULL_REQUEST_TEMPLATE:9]:
```suggestion
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`
```

Comment (jemten) [reply .github/PULL_REQUEST_TEMPLATE:9]: Do we need both front-end and back-end on clinical-db? Any thoughts @patrikgrenfeldt

Comment (patrikgrenfeldt) [reply .github/PULL_REQUEST_TEMPLATE:11]: There is only one update-script on clinical-db, it installs both frontend and backend so I think it should be sufficient with one

**Review by patrikgrenfeldt — APPROVED** [review]
👍

**Review by jemten — APPROVED** [review]

**Comment by henrikstranneheim** [comment]
Not deployed as this is only instructions

**Pull request merged**